qa/L0_sequence_batcher/sequence_batcher_test.py

```diff
@@ -40,20 +40,32 @@
 import sequence_util as su
 from tensorrtserver.api import *
 
-_test_system_shared_memory = bool(int(os.environ.get('TEST_SYSTEM_SHARED_MEMORY', 0)))
-_test_cuda_shared_memory = bool(int(os.environ.get('TEST_CUDA_SHARED_MEMORY', 0)))
-
-_no_batching = (int(os.environ['NO_BATCHING']) == 1)
-_model_instances = int(os.environ['MODEL_INSTANCES'])
-
-if _no_batching:
-    _trials = ("savedmodel_nobatch", "graphdef_nobatch", "netdef_nobatch", "plan_nobatch", "onnx_nobatch")
+TEST_SYSTEM_SHARED_MEMORY = bool(int(os.environ.get('TEST_SYSTEM_SHARED_MEMORY', 0)))
+TEST_CUDA_SHARED_MEMORY = bool(int(os.environ.get('TEST_CUDA_SHARED_MEMORY', 0)))
+BACKENDS = os.environ.get('BACKENDS', "graphdef savedmodel netdef onnx libtorch plan custom")
+ENSEMBLES = bool(int(os.environ.get('ENSEMBLES', 1)))
+
+NO_BATCHING = (int(os.environ['NO_BATCHING']) == 1)
+MODEL_INSTANCES = int(os.environ['MODEL_INSTANCES'])
+
+_trials = ()
+if NO_BATCHING:
+    for backend in BACKENDS.split(' '):
+        if (backend != "libtorch") and (backend != 'custom'):
+            _trials + (backend + "_nobatch",)
 elif os.environ['BATCHER_TYPE'] == "VARIABLE":
-    _trials = ("savedmodel", "graphdef", "netdef", "onnx")
+    for backend in BACKENDS.split(' '):
+        if (backend != "libtorch") and (backend != 'custom') and (backend != 'plan'):
+            _trials + (backend,)
 else:
-    _trials = ("custom", "savedmodel", "graphdef", "netdef", "plan", "onnx")
+    _trials = BACKENDS.split(' ')
+
 # Add ensemble to the _trials
-ENSEMBLE_PREFIXES = ["simple_", "sequence_", "fan_"]
+if ENSEMBLES:
+    ENSEMBLE_PREFIXES = ["simple_", "sequence_", "fan_"]
+else:
+    ENSEMBLE_PREFIXES = []
```

Comment:
I am under the impression that capitalized variables are constant or environment variables, so you should keep `ENSEMBLE_PREFIXES` as it is, and condition on whether testing ensemble model at line 74, i.e. `if ENSEMBLES: append the ensemble trials to _trails`.

Reply:
In this case ENSEMBLE_PREFIXES is not an environment variable. I can do what you said above but I will also have to modify all the other places ENSEMBLE_PREFIXES is used to check if ENSEMBLES is True.

Reply:
You don't need to check other places, if you take a closer look on how other places use `ENSEMBLE_PREFIXES`